### ISSUE TITLE
Suggestion: Instagram and Pinterest connect

### ISSUE THREAD

**Issue by msvitek** [QualityUnit/pap-public#53]
Customer XSB-TSNCC-896 would like an easy to use share button for Instagram and Pinterest for Promotion->Banner&links section.

**Comment by mkenderaqu** [comment]
It is not possible to create Instagram share button, because instagram API doesn't allow to add content via API.
```
At this time, uploading via the API is not possible. We made a conscious choice not to add this for the following reasons:

    Instagram is about your life on the go – we hope to encourage photos from within the app.
    We want to fight spam & low quality photos. Once we allow uploading from other sources, it's harder to control what comes into the Instagram ecosystem. All this being said, we're working on ways to ensure users have a consistent and high-quality experience on our platform.

```
https://instagram.com/developer/deprecated/endpoints/media/
https://instagram.com/developer/endpoints/media/

**Comment by mkenderaqu** [comment]
Pinterest share button is implemented, will be added to next release.

**Comment by lnikodem** [comment]
Pinterest share button implemented in release 5.4.24
